### ISSUE TITLE
Group the docs of similar methods in stdtypes.rst

### ISSUE THREAD

**Issue by ezio-melotti** [python/cpython#67748]
BPO | [23560](https://bugs.python.org/issue23560)
--- | :---
Nosy | @rhettinger, @ezio-melotti, @berkerpeksag, @vadmium, @csabella
PRs | <li>python/cpython#1709</li>
Files | <li>[str-groups.patch](https://bugs.python.org/file39506/str-groups.patch "Uploaded as text/plain at 2015-05-26.13:10:16 by @vadmium"): Groups str() methods</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2015-03-02.09:28:10.370>
labels = ['easy', 'type-feature', 'docs']
title = 'Group the docs of similar methods in stdtypes.rst'
updated_at = <Date 2019-08-09.02:38:23.499>
user = 'https://github.com/ezio-melotti'
```

bugs.python.org fields:
```python
activity = <Date 2019-08-09.02:38:23.499>
actor = 'Philip Dye'
assignee = 'docs@python'
closed = False
closed_date = None
closer = None
components = ['Documentation']
creation = <Date 2015-03-02.09:28:10.370>
creator = 'ezio.melotti'
dependencies = []
files = ['39506']
hgrepos = []
issue_num = 23560
keywords = ['patch', 'easy']
message_count = 5.0
messages = ['237036', '244101', '294049', '294080', '294117']
nosy_count = 7.0
nosy_names = ['rhettinger', 'ezio.melotti', 'docs@python', 'berker.peksag', 'martin.panter', 'cheryl.sabella', 'Philip Dye']
pr_nums = ['1709']
priority = 'normal'
resolution = None
stage = 'patch review'
status = 'open'
superseder = None
type = 'enhancement'
url = 'https://bugs.python.org/issue23560'
versions = ['Python 2.7', 'Python 3.5', 'Python 3.6']
```

</p></details>

**Comment by csabella** [comment]
Hi Martin,

Do you think it would be good to move forward with this as a PR?


Thanks!

**Comment by rhettinger** [comment]
I don't think this should be done.  Instead, add a table at the top listing the functions in groups, possibly linking to the alphabetical details docs below.

When looking a what a string method does, alphabetical makes the search easier than having to guess which group something is in before you can find it.

**Comment by csabella** [comment]
I hope this is OK, but I created a version of the patch with a table as Raymond suggested.  I didn't know how to make a patch, so I did it in a PR.  I wanted to have a comparison available to see the differences between the two versions.  I used the original categories as defined by Martin.

I could also add a bytearray column if the table looks reasonable.

Thanks!

**Comment by OTheDev** [comment]
@encukou Can this issue be closed?
